4.plot_mutation_results.ipynb

```diff
@@ -0,0 +1,494 @@
XuMk4ej1cKFMkTKJKmlJ7XMoYU9f1Feu+ef5Xl9ci7YGmiLdKb/G2iPVKkVybam/h9tlPxgfZo4vcoXkGiYE8p1pNF/PBpvmEFfrLwT90x0P8Y7/nvxR/gxtDDC+FmNiJvvqC+qNH/vD9PO/ceVXdvVJ9dMEc3X/tRXdT+qqSjki6+oOcAzhffqZzhv/waTZ05VT3vfZD4bLK8Sv3anJ27Xi/M0B0PMYV3PiR/IOB2t84Z2UCucuxCwnfffbcaGhr0wx/+UMXFxWpsTJxQqK6uTvX19Vq0aJFTXQGyipvZONMdUajjrI53hnW8M6wTnd063hlW28mz8vu8+tqKyqw94pvOkKJeiaLefHK57dHhc2FMXIpGZCI9UrRXJrWgHyjiB6ZFhxX7A4/pOSMT6UjM1/+Y1Ev2fCjLJ49vsHhPe5R+WJE/UNR7/IWJ/li5dfkft7Jx5LV92nZstk5F39OiS+Zo+XWf1MVzLpIkhff8VlLmHVrAeOM7FcbD8B0PgcrFic9Od7s1KmQDuchjjM34lBzGEBKGkLg99DBbDc/GgYMn9Y9bDyTvW16PZk2fpDklRYrGjD5+yTQt+4xzR8fHW7Zlw8SiaQr8lGI+9Qj+kNuDbak7ABTtzfic3knFmrxm89BpZGNENo78/G6VhA8PnSlQJE/BZJmeDySPV765VyR2+gQmyRMokicwSZ7ApP6dQYPTVNB/3zty//fApQXdLu6zLRtu4HPDHt+pyAbZsEc2yEbODVkHkH3mB4u1prpCJVMLNKe0SDMuKpTlza0jqLnMY/kSR74LJo/ZMhNH8/tSCvbeZMHe+79PyFc0ds+Vz2YUB6TCeQpcuSIxuqHnjEzvWZneM4q+cyBxIsNTbf3/EhEeeobmdKzAiKI91nFEHo+lWPv/ST6/PJZfsvzy+BK/E7cDid+WX/IFkvOkzjfQJq9v1CNauLQTAADuoiAHJqjiooAWV811uxsYQx6Pt38Ie+GItr7XWuT18Sf/XAQ+fp0kyX/pn4xoCz9zvySp6IZ1yWkmHpX6elJOPth/gsEhJyQcfoLCsBTpkzFxRQ7ulWIRKRrR4HWSz5NtIe+3Leo9ll/Rdw4oYlmyLj082JZa+Cd3FARSdgAEhi3TbzsCAAAAfDg+QQFgAvBX/GnO/S+gWzINIfdX/OmIaR6vTyqcIk/h6IapDR+yboxJnEwpGpGJRfqL9L7kbRONSLH++9H+af23E/P09c8z9PbgsiKJnQGpbd1diskotr9FF7QzwOMderTe5gh++iL/w+YPDBb+w3Y4yPJf8HkuGCUAAHATBTkATACpZwzG+RvL//ceviyPxyN5fFLAJ8/gleTHVfiZ++X3++Sr/k7iRIMZC/++ob9TdwaM2HEw8Li+/h0B4WE7DyIy0b7EfPHYhb0IyzfkyL2nv8C3K+DtRgxE3vit4pOnqmAlBTkAwHkU5AAATFDJkRMez+ARZ4evcmTi8cECvr9IT1v4J4/uZ9hhkFxONDGtr1tmxLL7l9s/KiAa61OBsy8bAABJFOQAAExY2TBywuP1St6CxOX6HHxeY4wUjyrc0phT11oGAOQXCnIAADDhePpHBeTitZYBAPmDghwAAExY2TBKAAAwcXHRYQAAAAAAXEBBDgAAAACACxiyDgCY0KZ+4y+l48eT93tX3qye2+qkcFgXrf7zEfP33PJl9d7yZXk6OlT8tTUj2//ia+q9aZW87x7T1G9+Y0R791//jfqql8l6+y1N+c7fjmgPr71DkesWy/r9AU35+4YR7Wf/br2iV39Gvn0vavJ9G0a0n9n4gGKLPiH/b/5bRZseHNn+Dw8rdullCjy7U5N+tEXyW7ooMnjpsQ+aHlV87jwVNG9T4b/8ZMTju37yuExpqQqe/IUKn/zFiPb3/+3fpaIiFT72Tyr4j6dHtjfvkCRNatqswH/uGtpYWKj3n3xKklT0g0b59/xmSLOZXqKun/5ckjT53rvle3nfkPZ4sEwf/OifE+13fle+138/pD32sUt15gebJUlTbq+XdfDtREP/OoguXKSz9zYm2u9skH70yIj+AwAwlijIAQAAMAI7q9hZxc4qYPxRkAMAJrQPHv2p4nEzsqGoKPkF2Y4pLc3YHp87L2N77NLLMrcv+kTG9ujVn8nYHrlusd6/bnHa9r7qZeqrXqaZM6fqfZsTmvXetEq9N61K+/je/uIrnZ7b6hLFWxrd36xX9zfr07aHb/+udPt307afvfPutG2SksVDOgPFhyTbdXDm3ge4NjkAYNx5jDE230JyV0fHGfsvVhPEzJlT9d4EP1Os3Trwej0qLZ3iUo+yA9kgG2TDHtkgG2TDHtkgG2TDHtkgG2OZDU7qBgAAAACACyjIAQAAAABwAQU5AAAAAAAuoCAHAAAAAMAFFOQAAAAAALiAghwAAAAAABc4dh3yQ4cOqaGhQadPn9a0adPU2Nio+fPnD5mnqalJO3bskGVZ8vl8Wrt2ra699lqnugi4gmwA9sgGYI9sAPbIBnKRY9ch/8pXvqJVq1bpxhtv1K9+9Stt27ZNP/vZz4bMs2fPHn3qU5/SpEmT9MYbb+jWW2/V888/r8LCwnN+Hq4LyHUBc+2amWTDGWSDbKRDNsgG2bBHNsgG2bBHNshGzl2HvKOjQ62traqpqZEk1dTUqLW1VZ2dnUPmu/baazVp0iRJUkVFhYwxOn36tBNdBFxBNgB7ZAOwRzYAe2QDucqRIeuhUEizZ8+WZVmSJMuyNGvWLIVCIZWUlNg+prm5WR/5yEc0Z86cUT1Xtu6xc9LMmVPd7oLrcmUdkA1n5cr7YjzlyjogG87KlffFeMqVdUA2nJUr74vxlCvrgGw4K1feF+NprNaBY/9DPhr79u3Tww8/rMcee2zUj2UICUNIcm141WiQjfNHNshGOmSDbJANe2SDbJANe2SDbOTckPVgMKgTJ04oFotJkmKxmNrb2xUMBkfMu3//ft1xxx1qampSeXm5E90DXEM2AHtkA7BHNg
```

Comment:
Adding a legend at the end to explain what the different lines mean here would be useful for interpreting the plots